items.json

```diff
@@ -10543,43 +10543,43 @@
         "accuracy": 0,
         "missileSpeed": 0,
         "stackAmount": 0,
-        "length": 208,
+        "length": 201,
         "balance": 0.0,
-        "handling": 68,
+        "handling": 65,
         "bodyArmor": 0,
         "flags": [
           "MeleeWeapon",
           "WideGrip"
         ],
-        "thrustDamage": 27,
+        "thrustDamage": 25,
         "thrustDamageType": "Pierce",
-        "thrustSpeed": 90,
-        "swingDamage": 45,
+        "thrustSpeed": 88,
+        "swingDamage": 42,
         "swingDamageType": "Cut",
-        "swingSpeed": 34
+        "swingSpeed": 25
       },
       {
         "class": "TwoHandedPolearm",
         "itemUsage": "polearm_block_long_shield_swing_thrust",
         "accuracy": 0,
         "missileSpeed": 0,
         "stackAmount": 0,
-        "length": 208,
-        "balance": 0.21,
-        "handling": 66,
+        "length": 201,
+        "balance": 0.06,
+        "handling": 62,
         "bodyArmor": 0,
         "flags": [
           "MeleeWeapon",
           "NotUsableWithOneHand",
           "WideGrip",
           "TwoHandIdleOnMount"
         ],
-        "thrustDamage": 27,
+        "thrustDamage": 25,

```

Comment:
you made it a pierce weapon. That's why you lack tier budget for swing. Reduce pierce to 16, or up it at 30 but change it to cut damage

Reply:
I changed it now its tier 9.7 and can stab and swing